opal/mca/memory/patcher/memory_patcher_component.c

```diff
@@ -591,6 +591,9 @@ static int patcher_open (void)
 
 #if defined (SYS_brk)
     rc = opal_patcher->patch_symbol ("brk", (uintptr_t)intercept_brk, (uintptr_t *) &original_brk);
+    if (OPAL_SUCCESS != rc) {
```

Comment:
rc will be returned in any case, here evaluation of returned value is not needed

Reply:
@hoopoepg ok. I would like to support the code style. Should I close it?

Reply:
let's wait for another opinions :)

Reply:
The code was intentionally written this way. The return code is already passed up.